site/content/blog/carvelizing-helm-chart.md

```diff
@@ -0,0 +1,299 @@
+---
+title: "Carvelizing Helm Chart"
+slug: carvelize-helm-chart
+date: 2022-02-16
+author: Rohit Aggarwal
+excerpt: "Use Carvel to author and consume helm chart "
+image: /img/logo.svg
+tags: ['carvel', 'helm', 'gitops']
+---
+# Carvelizing Helm Chart
+
+## Introduction
+
+In this blog, we will see how to author [`bitnami nginx helm chart`](https://github.com/bitnami/charts/tree/master/bitnami/nginx) into Carvel package and then consume the same.
```

Comment:
This can move after line 20.  
reword to:
In this blog, we will see how to author a Carvel package from a helm chart (specifically [`bitnami nginx helm chart`](https://github.com/bitnami/charts/tree/master/bitnami/nginx) ).  We shall also then consume the same to install nginx.

Reply:
If we will move it down, then we should remove the `## Why should I choose Carvel`. Let's discuss more on it tomorrow